docs/hazmat/primitives/asymmetric/dsa.rst

```diff
@@ -402,6 +395,13 @@ Key interfaces
         :return bytes: Serialized key.
 
 
+.. class:: DSAPublicKeyWithSerialization
+
+    .. versionadded:: 0.8
+
+    Alias of :class:`DSAPublicKey`.
```

Comment:
I don't have a strong preference between those two, "Alias of" doesn't read
super great to me though.

On Sat, Jul 4, 2015 at 1:55 PM, Paul Kehrer <notifications@github.com>
wrote:

> In docs/hazmat/primitives/asymmetric/dsa.rst
> <https://github.com/pyca/cryptography/pull/2112#discussion_r33889049>:
>
> > @@ -402,6 +395,13 @@ Key interfaces
> >          :return bytes: Serialized key.
> >
> >
> > +.. class:: DSAPublicKeyWithSerialization
> > +
> > +    .. versionadded:: 0.8
> > +
> > +    Alias of :class:`DSAPublicKey`.
>
> I'm neutral so do you have a preference?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2112/files#r33889049>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084